package for_practice.Abstract_classes;

public class Main {
    public static void main(String[] args) {
        System.out.println("=== Абстрактнi класи =====");
        Appliance washingMachine = new WashingMachine();
        Appliance microwave = new Microwave();

        washingMachine.turnOn();
        microwave.turnOn();
        System.out.println();

        // ===== Інтерфейси =====
        System.out.println("=== Iнтерфейси =====");
        Playable guitar = new Guitar();
        Playable piano = new Piano();

        guitar.play();
        piano.play();
    }
}